acid/features/status/tests/test_status.py

```diff
@@ -160,8 +160,8 @@ def test_remaining_with_job_wo_remaining_should_return_none(self, job,
         assert buildset.remaining_time is None
 
     def test_remaining_wh_multiple_jobs_should_return_longest_time(
-        self, buildset, job_factory):
-        test_jobs = [job_factory() for x in range(3)]
+        self, buildset, many_jobs):
```

Comment:
Add an extra level of indentation for function arguments. For a reference check [PEP8](https://www.python.org/dev/peps/pep-0008/#indentation).